### DISCUSSION TITLE
24/7 viewer?

### DISCUSSION THREAD

**Discussion by bairhys** [blakeblackshear/frigate#9111] [Ask A Question] [answered]
Does Frigate have a 24/7 recording viewer? How are others viewing 24/7 recordings when there is no detected event (maybe a samba share)?

I have Frigate recording continuously but cannot find how to view the continuous recordings in Frigate web ui. See below a `ls` a few folders deep inside my Frigate `recordings` folder. 

```
$ ls
00.03.mp4  01.13.mp4  02.23.mp4  03.33.mp4  04.43.mp4  05.53.mp4  07.03.mp4  08.13.mp4  09.23.mp4
00.13.mp4  01.23.mp4  02.33.mp4  03.43.mp4  04.53.mp4  06.03.mp4  07.13.mp4  08.23.mp4  09.33.mp4
00.23.mp4  01.33.mp4  02.43.mp4  03.53.mp4  05.03.mp4  06.13.mp4  07.23.mp4  08.33.mp4  09.43.mp4
00.33.mp4  01.43.mp4  02.53.mp4  04.03.mp4  05.13.mp4  06.23.mp4  07.33.mp4  08.43.mp4  09.53.mp4
00.43.mp4  01.53.mp4  03.03.mp4  04.13.mp4  05.23.mp4  06.33.mp4  07.43.mp4  08.53.mp4  10.03.mp4
00.53.mp4  02.03.mp4  03.13.mp4  04.23.mp4  05.33.mp4  06.43.mp4  07.53.mp4  09.03.mp4  10.13.mp4
01.03.mp4  02.13.mp4  03.23.mp4  04.33.mp4  05.43.mp4  06.53.mp4  08.03.mp4  09.13.mp4
```

**Comment by NickM-27** [comment, marked as answer]
press the `recordings` button on any camera and it will open the 24/7 recordings viewer where you can select an hour or jump to an event within that hour

**Comment by bairhys** [comment]
Thanks for your help, it works well! I didn't try clicking the hour earlier.

For anyone else that finds this, on the camera recording page, need to click on these hour heading buttons and the entire continuous hour recording starts playing.

![image](https://github.com/blakeblackshear/frigate/assets/33058425/b5bf55f2-23fe-4491-9b0a-50e5d9913da2)

Or when you click on an event on the recordings page, the events hour starts playing, skipping to the start of the event. This allows viewing more footage before and after an event.

**Reply by NickM-27** [reply]
The UI is being rewritten for 0.14 and it will be easier to view recordings
